src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+let total = 0;
+let average = 0;
+
+const numbers = [...document.querySelectorAll('span.population')];
+
+const population = numbers.map((item) => item.textContent.replaceAll(',', ''));
+
+for (const number of population) {
+  total += Number(number);
+}
+
+average = total / population.length;
```

Comment:
Ensure that 'population.length' is not zero to avoid division by zero, which would result in Infinity for the average.